lib/bank_accounts/savings_account.rb

```diff
@@ -0,0 +1,21 @@
+module BankAccounts
+
+  class SavingsAccount < Account
+
+    # ADDS INTEREST TO BALANCE & RETURNS INTEREST AMOUNT
+    def add_interest(rate)
+      interest = @balance * (rate/100)
+      @balance += interest
+      return interest
```

Comment:
Since Ruby has an implicit return of the last line of the method, you don't need to say `return`.